README.md

```diff
@@ -69,6 +80,15 @@ struct DemoAppApp: App {
                                                ignoreErrors: [],
                                                customDomainUrl: "",
                                                labels: ["String" : Any],
+                                               sampleRate: 100,
+                                               mobileVitalsFPSSamplingRate: 300, // minimum every 5 minute
+                                               instrumentations: [.navigation: true,
+                                                                  .mobileVitals: true,
+                                                                  .custom: true,
+                                                                  .errors: true,
+                                                                  .userActions: true,
+                                                                  .network: true,
+                                                                  .anr: true],
```

Comment:
as we discussed, the main init should be without many config
each "feature" gets its own section